content/03.results.md

```diff
@@ -296,5 +297,7 @@ This result may be explained by the higher expression of _CD274_ found in WNT su
 
 Finally, we asked whether any molecular subtypes might have a high ratio CD8+ to CD4+ T cells, a metric which has been associated with better immunotherapy response and prognosis following PD-L1 inhibition in non-small cell lung cancer or adoptive T cell therapy in multiple stage III or IV cancers [@doi:10.1136/jitc-2021-004012; @doi:10.4236/jct.2013.48164].
 While adamantinomatous craniopharyngiomas and Group 3 and Group 4 medulloblastomas had the highest CD8+ to CD4+ T cell ratios (**Figure {@fig:S6}F**), very few tumors had ratios greater than 1, highlighting an urgent need to identify novel therapeutics for pediatric brain tumors with poor prognosis.
+To explore the potential influence of tumor purity, selected transcriptomic analyses were repeated using samples with tumor purities at or above the median tumor purity of their cancer group (see **STAR Methods**).
+The analyses using all stranded samples were broadly consistent (**Figure {@fig:S7}D-I**) with those using samples with high tumor purity, noting that the latter results show tighter and less disperse UMAP clustering. 
```

Comment:
I don't think that we should be talking about the distances between clusters or cluster size in a UMAP because that might not be meaningful.

```suggestion
The analyses using all stranded samples were broadly consistent (**Figure {@fig:S7}D-I**) with those using samples with high tumor purity. 
```

Reply:
I suppose we can leave that out and generalize but check out Reviewer 2's comment wanting us to note that when we "control for" purity, our results are tighter and used umap as the tangible example

Reply:
Yes, again, I know why you put this here. I do not agree with the  interpretation.

Reply:
ok got it